sale_order_line_input/models/sale_order.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2018 Tecnativa - Carlos Dauden
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    pricelist_id = fields.Many2one(
+        related="order_id.pricelist_id",
+        readonly=True,
```

Comment:
This is the default

Reply:
@rousseldenis Odoo only adds pricelist_item_id on model sale.order.line, we should add pricelist_id
@pedrobaeza

Reply:
@carolinafernandez-tecnativa I know. What I wnated to say, readonly attribute is True by default on related fields. So, no need to declare it.

Reply:
@rousseldenis done!